docs/troubleshooting_pi-server_integration.rst

```diff
@@ -0,0 +1,180 @@
+
+Possible solutions to common situations
+---------------------------------------
+
+- **Recreate a single or a sets of PI-Server objets and send again all the data for them to the PI Server on the Asset Framework hierarchy level**
+    procedure:
+        - disable the 1st north instance
+        - delete properly the objects on the PI Server, AF + Data archive, that we intend to recreate or were eventually partially deleted
+        - create a new **DISABLED** north instance using a new/unique name and having the same AF hierarchy of the 1st north instance
+        - install *fledge-filter-asset* on the new north instance, available at https://github.com/fledge-iot/fledge-filter-asset
+        - configure *fledge-filter-asset* with a rule like the following one
+        - enable the 2nd north instance
+        - let the  2nd north instance send the desired amount of data and then disable it
+        - enable the 1st north instance
+
+    consideration:
+        - the 2nd north instance will be used only to recreate the objects and resend the data
+        - the 2nd north instance will send again all the data available for the specified *included* assets
+        - there will some data duplicated for the recreated assets because parte of the information will be managed by both the north instances.
+
+.. code:: json
+
+    {
+      "rules": [
+        {
+          "asset_name": "asset_4",
+          "action": "include"
+        }
+      ],
+      "defaultAction": "exclude"
+    }
+
+- **Recreate all the PI-Server objets and send again all the data to the PI Server on a different Asset Framework hierarchy level**
+    procedure:
+        - disable the 1st north instance
+        - create a new north instance using a new/unique name and having a new AF hierarchy (North option 'Asset Framework hierarchies tree')
+
+    consideration:
+        - this solution will create a set of new objects unrelated to the previous ones
+        - all the data stored in Fledge will be sent
+
+- **Recreate all the PI-Server objets and send again all the data to the PI Server on the same Asset Framework hierarchy level of the 1st North instance WITH data duplication**
+    procedure:
+        - disable the 1st north instance
+        - delete properly the objects on the PI Server, AF + Data archive, that were eventually partially deleted
+        - stop / start PI Web API
+        - create a new north instance 2nd using the same AF hierarchy (North option 'Asset Framework hierarchies tree)
+
+    consideration:
+        - all the types will be recreated on the PI-Server and if the structure of each asset, number and types of the properties, is always the same and never changes everything should work properly as PI Web API 2019 SP1 1.13.0.6518 is not going to complain with it
+        - PI Web API 2019 SP1 1.13.0.6518 creates PI-Server objects having the compression disabled, so all the objects already present in the Data Archive will have duplicated data
```

Comment:
Using PI Web API 2019 SP1 1.13.0.6518 the PI-Server creates objects with the compression feature disabled. This will cause any data that was previously loaded and is still present in the Data Archive,  to be duplicated.

Reply:
done